src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -329,7 +329,20 @@ int mongoSubCacheItemInsert
 
   if (stringFilterP != NULL)
   {
-    cSubP->expression.stringFilter = *stringFilterP;
+    std::string errorString;
+
+    if (!cSubP->expression.stringFilter.fill(stringFilterP, &errorString))
+    {
```

Comment:
Maybe is better to remove the empty { } block and put the comment above the if. Makes sense?